### PR TITLE
New functions: alm algebraic operations and indexing

### PR DESCRIPTION
**Pull request by LeeoBianchi**

### CODE REVIEW

File: docs/src/alm.md [hunk 1]
```diff
@@ -118,16 +112,75 @@ pixwin
...
+#will return a new object:
+myalm*myf_l
+myalm/myf_l
+
+#will overwrite myalm:
+myf_l*myalm
+myf_l\myalm #here the inverse division must be used
+```
+
+In either case the call to such operator consists in a shortcut to the function [`almxfl`](@ref) (or [`almxfl!`](@ref), if in place).
```
Comment (ziotom78) [inline docs/src/alm.md:157]: This is counter-intuitive and hard to remember; better to remove in-place operators and force people to use explicit functions instead. (This is also suggested in the [Julia forum](https://discourse.julialang.org/t/overload-addition-assignment-operator/28836/13?u=maurizio_tomasi).)

**Comment by LeeoBianchi** [comment]
@ziotom78 I've pushed a couple of simple functions I had to code up for my work which I think could be useful for a Healpix user in general.

With this I think the PR can be merged, if you're happy with it.

Btw, having all these new functions included in a new Healpix.jl version would make things much easier for the release of my new package :)

**Comment by ziotom78** [comment]
> @ziotom78 I've pushed a couple of simple functions I had to code up for my work which I think could be useful for a Healpix user in general.
> 
> With this I think the PR can be merged, if you're happy with it.
> 
> Btw, having all these new functions included in a new Healpix.jl version would make things much easier for the release of my new package :)

Great! I'm fine with these changes, so we can merge the PR. But can you please add a entry in the CHANGELOG listing the functions that this PR adds?

**Comment by LeeoBianchi** [comment]
> Great! I'm fine with these changes, so we can merge the PR. But can you please add a entry in the CHANGELOG listing the functions that this PR adds?

I added one last function to get the pixels corresponding to a specific ring in a map, which I had in my scripts and thought could be useful when operating with HealpixMaps. I also updated the CHANGELOG.md.

Everything should be in place now!

**Pull request merged**